mods/common/chrome/assetbrowser.yaml

```diff
@@ -203,3 +206,4 @@ Background@ASSETBROWSER_PANEL:
 			Height: 25
 			Font: Bold
 			Text: Close
+		TooltipContainer@TOOLTIP_CONTAINER:
```

Comment:
Please keep the end-of-file newline here.

Reply:
EOF lines fixed everywhere.